src/main/java/org/triplea/client/ui/javafx/TripleA.java

```diff
@@ -0,0 +1,105 @@
+package org.triplea.client.ui.javafx;
```

Comment:
> because you then know what expects you inside the poackage even if you don't know what javafx is

I think I understand your reasoning here.  `ui` is simply to tag the subpackages as containing pure UI classes for a specific framework?

You've obviously thought out the JFX client way beyond the state of this PR, so can you provide some examples of other client-specific packages you see being added in the future (e.g. for client-specific models, utilities, etc.).  I know it's not part of this PR, but if you can throw in some theoretical examples for the Swing client, as well, that might help to give me a better picture of how the client packages should be structured.  I ask that because I'm wondering if you see client-specific code being shared between the Swing and JFX clients or if they are going to be totally separate (obviously they will both reference core "non-client" code, such as the engine, etc.).

Note that I'm ok with the package name as-is.  I think it addresses @DanVanAtta's primary concern.  I want to have the discussion above, but I don't want it to hold up merging this PR.  We can always rename later, if necessary.  I just want to make sure everyone has the same expectations for the package structure so that there are clear conventions about where things go, as opposed to some of the uncertainty we have today. :smile: